salt/report_upload/srv/salt/_runners/foreman_report_upload.py

```diff
@@ -3,6 +3,7 @@
 Uploads reports from the Salt job cache to Foreman
 '''
 from __future__ import absolute_import, print_function, unicode_literals
+from builtins import str
```

Comment:
Why would one need to import str?

Reply:
To get rid of the `unicode = str` line https://github.com/theforeman/smart_proxy_salt/blob/master/salt/report_upload/srv/salt/_runners/foreman_report_upload.py#L30-L31 that the linter has an issue with.

It is a python 2/3 compatibility fix: https://python-future.org/compatible_idioms.html#unicode

Reply:
I do not know that but there is still a lot of compatibility fixes in the code. 

The PR is not about removing python 2 compatibility but we can do that in a separate commit.

Reply:
Ok

Reply:
`pyupgrade --py36-plus --keep-percent-format `find . -name "*.py" -type f`` should do the job :D